TOC.md

```diff
@@ -0,0 +1,48 @@
+# TiDB 中文开发者指南
+
+## 目录
+
++ 安装 MySQL 客户端
++ 连接 TiDB 数据库
+  + 命令行工具
+    + MySQL Client
+  + 图形化工具
+    + DBeaver
+    + Navicat
+    + PHPMyAdmin
+    + Workbench
+  + DRIVERS
+    + Java
+    + C++
+    + Go
+    + Python
+    + JavaScript
++ 创建和管理数据库对象
+  + [创建和管理库](database-management/.md)
+  + [创建和管理表](table-management/.md)
+  + 创建和管理分区表
+  + 创建和管理视图
+  + [创建和管理索引](index-management/.md)
++ [TiDB 事务说明](transaction/.md)
++ 基本 DML 语句说明
+  + 写入数据
+  + 更新数据
+  + 删除数据
+  + 查询数据
++ 错误处理及故障排查
+  + [故障诊断](troubleshooting/.md)
+  + [使用 TiFlash](troubleshooting-tiflash/.md)
++ [性能优化及建议](performance-optimization/.md)
++ [数据迁移](data-migration/.md)
```

Comment:
All wrong links. Please be more careful.

Reply:
updated